mne_bids/read.py

```diff
@@ -495,6 +496,7 @@ def get_head_mri_trans(bids_path):
     if ext == '.fif':
         extra_params = dict(allow_maxshield=True)
 
+    # make sure reading in the MEG file
```

Comment:
I don't know what this comment is supposed to tell me

Reply:
oops leftover...